src/WordPress/DataStore.php

```diff
@@ -120,18 +117,18 @@ public function setPluginSetting($settingId, $value)
             return false;
         }
 
-        return update_option(self::CLOUDFLARE_SETTING_PREFIX.$settingName, $value);
+        return update_option($settingName, $value);
     }
 
     private static function getPluginSettingName($settingId)
```

Comment:
If you move `$settings` from `getPluginSettings()` into a class variable you can just do:
```
return array_key_exists($settingId, $settings);
```